source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.tsx

```diff
@@ -212,12 +212,7 @@ export const AcquisitionContainer: React.FunctionComponent<IAcquisitionContainer
   };
 
   const canRemove = async (propertyId: number) => {
-    const fileProperties = await retrieveAcquisitionFileProperties(acquisitionFileId);
-    const fp = fileProperties?.find(fp => fp.property?.id === propertyId);
-    return (
-      fp?.activityInstanceProperties?.length === undefined ||
-      fp?.activityInstanceProperties?.length === 0
-    );
+    return true;
```

Comment:
could you please explain why we removed the existing logic?

Reply:
There are not activity instances anymore. This should have been removed once we removed the activities.